handsontable/src/shortcuts/context.js

```diff
@@ -0,0 +1,153 @@
+import { createUniqueMap } from '../utils/dataStructures/uniqueMap';
+import { normalizeKeys } from './utils';
+import { isUndefined } from '../helpers/mixed';
+
+/**
+ * Create shortcuts' context.
+ *
+ * @param {string} name Context's name.
+ * @returns {object}
+ */
+export const createContext = (name) => {
+  const SHORTCUTS = createUniqueMap({
+    errorIdExists: keys => `The passed keys combination "${keys}" is already registered in the "${name}" context.`
+  });
+
+  /**
+   * Add shortcut to the context.
+   *
+   * @param {Array<Array<string>>} variants Shortcut's variants.
+   * @param {Function} callback The callback.
+   * @param {object} [options] Additional options for shortcut's variants.
```

Comment:
I'm not sure if the `options` argument should be marked as optional if `options.namespace` is required.

Reply:
API for that method has been changed within https://github.com/handsontable/handsontable/pull/8955/commits/2e51b7d36712f9d0adcf266b4b689baeae86e248. There is no sense to separate `variants` and `callback` settings from other options. The config is now an object.